test/cypress/cypress/e2e/routeTest.cy.js

```diff
@@ -0,0 +1,248 @@
+
+beforeEach(() => {
+  cy.visit("localhost:3000");
+});
+
+
+
+describe('showing the basics of site are up', () => {
+
+    it('title showing', () => {
+      cy.title().should('eq', 'AI Daily');
+      cy.url().should('include', 'localhost:3000')
+        })
+
+    it('News sections showing', () => {
+        cy.contains('NEWS').click()
```

Comment:
There is a little bit of code repetition in the tests, would it be possible to create a single function that is called for each test?
Perhaps using the section title (eg 'NEWS'), and anything else that changes as arguments.